### ISSUE TITLE
What's the correct setup for irbrc?

### ISSUE THREAD

**Issue by jrhorn424** [janlelis/irbtools-more#1]
This repository states the following is sufficient to use irbtools-more:

    require 'irbtools/more'

However, the main irbtools repo suggests using:

    require 'irbtools/configure'
    Irbtools.add_package :more
    Irbtools.start

Are these equivalent? Should there be one, canonical way to use this in a non-Windows environment?

It also isn't clear whether I need *both* or just the second of the following in my Gemfile:

    gem 'irbtools', :require => false
    gem 'irbtools-more', :require => false

In other words, shound installing (or upgrading) the latest irbtools-more gem also install the latest irbools gem as a prerequisite? The gemspec seems to indicate so, but I'm new to Ruby and Rails.
